log10/load.py

```diff
@@ -68,67 +66,57 @@ def _func_with_backoff(func, *args, **kwargs):
     return _func_with_backoff(func, *args, **kwargs)
 
 
-# todo: should we do backoff as well?
-def post_request(url: str, json_payload: dict = {}) -> requests.Response:
-    headers = {"x-log10-token": token, "Content-Type": "application/json"}
-    json_payload["organization_id"] = org_id
-    try:
-        # todo: set timeout
-        res = requests.post(url, headers=headers, json=json_payload)
-        # raise_for_status() will raise an exception if the status is 4xx, 5xxx
-        res.raise_for_status()
-        logger.debug(f"HTTP request: POST {url} {res.status_code}\n{json.dumps(json_payload, indent=4)}")
-
-        return res
-    except requests.Timeout:
-        logger.error("HTTP request: POST Timeout")
-        raise
-    except requests.ConnectionError:
-        logger.error("HTTP request: POST Connection Error")
-        raise
-    except requests.HTTPError as e:
-        logger.error(f"HTTP request: POST HTTP Error - {e}")
-        raise
-    except requests.RequestException as e:
-        logger.error(f"HTTP request: POST Request Exception - {e}")
-        raise
-
-
-post_session_request = functools.partial(post_request, url + "/api/sessions", {})
+# TODO: Retries on 5xx errors has to be handled in user code, so recommendation is to use tenacity.
+transport = httpx.HTTPTransport(retries=5)
+httpx_client = httpx.Client(transport=transport)
 
 
-def get_session_id():
-    if target_service == "bigquery":
-        return str(uuid.uuid4())
-
+def post_request(url: str, json: dict = {}) -> requests.Response:
```

Comment:
```suggestion
def post_request(url: str, json: dict = {}) -> httpx.Response:
```
also remove the `import requests`